### PR TITLE
Document that shared dirs are on same space as user homedirs

### PR DESCRIPTION
**Pull request by yuvipanda**
fixes https://github.com/2i2c-org/infrastructure/issues/2992

[edited by sgibson91 to close the linked issue]

### CODE REVIEW

File: docs/topic/infrastructure/storage-layer.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ Note that `<username>` is specified as a `subPath` - the *subdirectory* **in the
 
 #### Shared directories
 
+Shared directories use the same storage as users' home directories, and are
+not size limited in any other way. If more space is required, resize the
+home directory storage up.
```
Comment (GeorgianaElena) [inline docs/topic/infrastructure/storage-layer.md:58]: Maybe also link to the docs describing how to increase it?

```suggestion
not size limited in any other way. If more space is required, [resize the
home directory storage up](https://infrastructure.2i2c.org/sre-guide/common-problems-solutions/#no-space-left-on-device-error).
```

**Review by GeorgianaElena — APPROVED** [review]

**Comment by sgibson91** [comment]
[Non-blocking comment] Are there other ways to diagnose this? We hadn't actually reached the "no space left on device error" message state. I also tried looking at (central) grafana but everything seems to be no data over there.

**Comment by yuvipanda** [comment]
@sgibson91 I'll work on figuring out how we can keep track of this!

**Review by consideRatio — APPROVED** [review]
The hardcoded https link could be made to a sphinx reference to not break as easily, but this is an incremental improvement as it is

**Pull request merged**